### PR TITLE
Add Ethereum Classic support

### PR DESCRIPTION
**Pull request by Har01d**
This PR proposes adding Ethereum Classic to 3xpl.

### CODE REVIEW

**Comment by Har01d** [comment]
@alexqrid, your comment is required regarding the geth fix we're using.

**Comment by alexqrid** [comment]
This modules will works with the node compiled with the following PR - https://github.com/3xplcom/ethereum-classic/pull/1 , i.e. it expects `status` field of the transactions receipts for `pre-Byzantine` blocks as well as for `post-Byzantine` blocks.

File: Modules/EthereumClassicMainModule.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        $this->first_block_date = '2015-07-30';
+        $this->currency = 'ethereum-classic';
+        $this->currency_details = ['name' => 'Ethereum Classic', 'symbol' => 'ETC', 'decimals' => 18, 'description' => null];
+
+        // EVMMainModule
+        $this->evm_implementation = EVMImplementation::geth;
+        $this->extra_features = [EVMSpecialFeatures::HasOrHadUncles];
+        $this->reward_function = function($block_id)
+        {
+            // https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1017.md
```
Comment (alexqrid) [inline Modules/EthereumClassicMainModule.php:27]: Should be replaced with the actual link https://ecips.ethereumclassic.org/ECIPs/ecip-1017

Comment (Har01d) [reply Modules/EthereumClassicMainModule.php]: Fixed in [7483500](https://github.com/3xplcom/Core/pull/67/commits/74835006bd39f879bb6787d965759901383726b1).

**Comment by Har01d** [comment]
The blockchain has been successfully fully processed: https://3xpl.com/ethereum-classic

**Review by alexqrid — APPROVED** [review]

**Pull request merged**